From c14768de0098a9dc26995bc06b50072fe87f1c9c Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Mon, 19 Jan 2026 03:36:30 -0800
Subject: [PATCH] Added bazel server shutdown

---
 oss_scripts/run_build.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/oss_scripts/run_build.sh b/oss_scripts/run_build.sh
index 8a75cc4..e282934 100755
--- a/oss_scripts/run_build.sh
+++ b/oss_scripts/run_build.sh
@@ -27,3 +27,6 @@ if [ -n "${AUDITWHEEL_PLATFORM}" ]; then
   PYTHON_MINOR_VERSION=$(python3 -c "import sys; print(sys.version_info.minor)")
   auditwheel repair --plat ${AUDITWHEEL_PLATFORM} --exclude "$SHARED_LIBRARY_NAME" -w . *cp3$PYTHON_MINOR_VERSION*.whl
 fi
+
+bazel clean --expunge
+bazel shutdown
\ No newline at end of file
-- 
2.39.3 (Apple Git-146)

