From 3f7e1adc15de46400f2729a5f5dad150ad103690 Mon Sep 17 00:00:00 2001
From: Jack Olivieri <jolivieri@anaconda.com>
Date: Tue, 7 May 2024 15:17:46 +0200
Subject: [PATCH 6/6] clean up after bazel temp files

---
 oss_scripts/run_build.sh | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/oss_scripts/run_build.sh b/oss_scripts/run_build.sh
index 019bf6c..e61e0a4 100755
--- a/oss_scripts/run_build.sh
+++ b/oss_scripts/run_build.sh
@@ -26,3 +26,9 @@ if [ -n "${AUDITWHEEL_PLATFORM}" ]; then
   PYTHON_MINOR_VERSION=$(python3 -c "import sys; print(sys.version_info.minor)")
   auditwheel repair --plat ${AUDITWHEEL_PLATFORM} --exclude "$SHARED_LIBRARY_NAME" -w . *cp3$PYTHON_MINOR_VERSION*.whl
 fi
+
+bazel clean --expunge
+bazel shutdown
+# this is needed because on many build systems the cache is actually under /root.
+# but this may not always be true/allowed, hence the or operation.
+rm -rf /root/.cache/bazel || true