diff --git a/oss_scripts/configure.sh b/oss_scripts/configure.sh
index 0000000..1111111 100755
--- a/oss_scripts/configure.sh
+++ b/oss_scripts/configure.sh
@@ -39,10 +39,10 @@ if python -c 'import tensorflow' &> /dev/null; then
 else
   echo 'Installing tensorflow.'
   if [[ "$IS_NIGHTLY" == "nightly" ]]; then
     pip install tf-nightly
   else
-    pip install tensorflow==2.18.0
+    pip install tensorflow==2.18.1
   fi
 fi

 # Copy the current bazelversion of TF.
