From 2421eba3a6352c3562deff1d4900ac6f7866bd11 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Mon, 19 Jan 2026 05:39:06 -0800
Subject: [PATCH] Remove setuptools hard pin

---
 oss_scripts/pip_package/BUILD                      | 13 ++++---------
 1 files changed, 4 insertions(+), 9 deletions(-)

diff --git a/oss_scripts/pip_package/BUILD b/oss_scripts/pip_package/BUILD
index 1434e54..af42019 100644
--- a/oss_scripts/pip_package/BUILD
+++ b/oss_scripts/pip_package/BUILD
@@ -6,16 +6,11 @@ package(default_visibility = ["//visibility:private"])
 
 licenses(["notice"])  # Apache 2.0
 
-compile_pip_requirements(
+# compile_pip_requirements(
+# Disabled for conda build - use existing lock files from upstream
+filegroup(
     name = "requirements",
-    extra_args = [
-        "--allow-unsafe",
-        "--build-isolation",
-        "--rebuild",
-    ],
-    generate_hashes = True,
-    requirements_in = "@release_or_nightly//:requirements.in",
-    requirements_txt = REQUIREMENTS,
+    srcs = []
 )
 
 py_binary(
