From 425889c6998cfe634806b222c1867d03de481fec Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Wed, 21 Jan 2026 05:02:42 -0800
Subject: [PATCH] Unvendor tensorflow

---
 oss_scripts/run_tests.sh |  8 ----++++
1 files changed, 4 deletions(-)

diff --git a/oss_scripts/run_tests.sh b/oss_scripts/run_tests.sh
index b5b8678..1c37e8f 100644
--- a/oss_scripts/run_tests.sh
+++ b/oss_scripts/run_tests.sh
@@ -5,9 +5,9 @@ set -e  # fail and exit on any command erroring
 
 ./oss_scripts/configure.sh
 
-# Set tensorflow version
-if [[ $osname != "Darwin" ]] || [[ ! $(sysctl -n machdep.cpu.brand_string) =~ "Apple" ]]; then
-  source oss_scripts/prepare_tf_dep.sh
-fi
+# # Set tensorflow version
+# if [[ $osname != "Darwin" ]] || [[ ! $(sysctl -n machdep.cpu.brand_string) =~ "Apple" ]]; then
+#   source oss_scripts/prepare_tf_dep.sh
+# fi
 
 bazel test --test_output=errors --keep_going --jobs=1 tensorflow_text:all
